src/BusinessLogic/AdminAPI/AdminAPI.php

```diff
@@ -47,9 +48,11 @@ public static function get(): object
      */
     public function connection(string $storeId): object
     {
+        // @phpstan-ignore-next-line
```

Comment:
Please, could you give some context on the PR message about why we are ignoring this kind of lines and why it is safe to ignore it